snomed/com.b2international.snowowl.snomed.exporter.server/src/com/b2international/snowowl/snomed/exporter/server/net4j/SnomedExportServerIndication.java

```diff
@@ -94,7 +94,7 @@
 	private Set<String> modulesToExport;
 	private Date deltaExportStartEffectiveTime;
 	private Date deltaExportEndEffectiveTime;
-	private String clientNamespace;
```

Comment:
I see the following filenames in the archive when I initiate an export through the REST api:

![image](https://user-images.githubusercontent.com/2010200/30649204-fa972060-9e1f-11e7-9d9c-e8890b44e090.png)

Reply:
The point is that this field is called clientNamespace not just by coincidence.